tests/test_shear.py

```diff
@@ -126,6 +150,16 @@ def test_time_of_day():
         bw.Shear.TimeOfDay(anemometers, heights, by_month=False, plot_type='12x24')
```

Comment:
tests are failing now for test_time_of_day and test_time_series. Can you investigate why this happens?